src/System.IO.FileSystem/tests/SetTime_str_dt.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Collections;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.IO.FileSystem.Tests
+{
+    public abstract class SetTime_str_dt : FileSystemTest
+    {
+        private readonly string NonExistantFileName = "nonexistant_file_name";
+        private readonly TimeSpan Accuracy = TimeSpan.FromSeconds(3);
```

Comment:
These are fields rather than constants; should they begin with an underscore and lower case?